flexget/plugins/metainfo/trakt_lookup.py

```diff
@@ -181,214 +180,250 @@ class PluginTraktLookup(object):
-                                           account=config.get('account'))
+                entry['trakt_collected'] = self.trakt.is_season_in_collection(self.__get_season(entry, session),
+                                                                              entry['title'])
             except LookupError as e:
                 log.debug(e)
-            else:
-                entry['trakt_watched'] = watched
         return entry
 
-    def lazy_user_ratings_lookup(self, config, style, entry):
-        """Does the ratings lookup for this entry and populates the entry fields."""
-        if style in ['show', 'episode', 'season']:
-            lookup = lookup_series
-            trakt_id = entry.get('trakt_show_id', eval_lazy=True)
-        else:
-            lookup = lookup_movie
-            trakt_id = entry.get('trakt_movie_id', eval_lazy=True)
+    def lazy_episode_collected_lookup(self, entry):
         with Session() as session:
-            lookupargs = {'trakt_id': trakt_id,
-                          'session': session}
             try:
-                item = lookup(**lookupargs)
-                if style in ['show', 'episode', 'season']:
-                    rating_style = style
-                    if style == 'show':
-                        rating_style = 'series'
-                    elif style == 'episode':
-                        rating_style = 'ep'
-                    elif style == 'season':
-                        rating_style = 'season'
-                    # fetch episode data if style is not series
-                    if style == 'episode':
-                        item = item.get_episode(entry['series_season'], entry['series_episode'], session)
-                    if style == 'season':
-                        item = item.get_season(entry['series_season'], session)
-                    rating = ApiTrakt.user_ratings(style, item, entry.get('title'),
-                                                   username=config.get('username'),
-                                                   account=config.get('account'))
-                    entry['trakt_{}_user_rating'.format(rating_style)] = rating
-                else:
-                    movie_rating = ApiTrakt.user_ratings(style, item, entry.get('title'),
-                                                         username=config.get('username'),
-                                                         account=config.get('account'))
-                    entry['trakt_movie_user_rating'] = movie_rating
+                entry['trakt_collected'] = self.trakt.is_episode_in_collection(self.__get_episode(entry, session),
+                                                                               entry['title'])
             except LookupError as e:
                 log.debug(e)
         return entry
 
+    def lazy_movie_collected_lookup(self, entry):
+        with Session() as session:
```

Comment:
all of these lookup method are almost identical in structure, maybe create a generic one and use that?